letsencrypt/files/renew_letsencrypt_cert.sh.jinja

```diff
@@ -7,3 +7,4 @@ COMMON_NAME="$1"
   {{ letsencrypt.config_dir.path }}/live/${COMMON_NAME}/privkey.pem \
   > {{ letsencrypt.config_dir.path }}/live/${COMMON_NAME}/fullchain-privkey.pem || exit 1
   chmod 600 {{ letsencrypt.config_dir.path }}/live/${COMMON_NAME}/fullchain-privkey.pem || exit 1
+{%- if letsencrypt.post_renew.cmds %}
```

Comment:
@dmaphy, you're missing the rest of the `if` block. It should be:

```yaml
{%- if letsencrypt.post_renew.cmds %}
  {%- for cmd in letsencrypt.post_renew.cmds %}
    {{ cmd }}
  {%- endfor %}
{%- endif %}
```

Reply:
A second of inattentiveness and some days full of work. Now fixed